provider/datagen/data/segmenter/sentence.toml

```diff
@@ -255,7 +255,7 @@ right = "Sep"
 
 [[tables]]
 # SB8
-name = "ATerm_Close_Sp_ParaSep"
+name = "ATerm_Close_Sp_CR"
```

Comment:
Question: `CR` is part of `ParaSep`. https://www.unicode.org/reports/tr29/tr29-39.html#Sentence_Boundary_Rules Could you help me understand why we separate "ATerm_Close_Sp_ParaSep" and "ATerm_Close_Sp_CR"? Is it because the `SB3` rules are changed below?

Reply:
Since moving to alias rule, if previous (left) state is ATerm_Close_Sp_CR and new (right) is LF, we should handle SB3 rule. If keeping ATerm_Close_Sp_ParaSep (that is for SB10), we cannot know whether last character of previous state is CR.

Reply:
Gotcha.  Thanks!